app-i18n/transifex-client/transifex-client-0.14.2.ebuild

```diff
@@ -18,7 +18,7 @@ KEYWORDS="~amd64 ~x86"
 IUSE="test"
 RESTRICT="!test? ( test )"
 
-BDEPEND="test? ( dev-python/mock[${PYTHON_USEDEP}] )"
+BDEPEND="test? ( <dev-python/mock-4.0[${PYTHON_USEDEP}] )"
```

Comment:
I hope this is patchable... has upstream been noted/asked about it?

Reply:
Seems we can easily patch it: https://github.com/gentoo/gentoo/pull/19057

Reply:
I also proposed to change that upstream: https://github.com/transifex/transifex-client/pull/314